chromium/utils/translate-en-messages.py

```diff
@@ -1,139 +1,119 @@
-'''
+"""
 Script:       translate-en-messages.py
 Version:      2024.5.14.1
-Description:  Translate msg's from en/messages.json to [[output_langs]/messages.json]
+Description:  Translate messages from en/messages.json to other language directories.
```

Comment:
The original description is useful because reader understands the structure immediately and know what to edit to customize